### ISSUE TITLE
Control buttons and presets broken in 1.5.0 Legacy on High Sierra (MBP A1502)

### ISSUE THREAD

**Issue by jfoster** [crystalidea/macs-fan-control#217]
When the control buttons are pressed they are not highlighted or selected. This is also true of the "Change control" option within the menubar item which does not display any window.

<img width="933" alt="Screen Shot 2019-10-04 at 12 33 38" src="https://user-images.githubusercontent.com/8654883/66204838-b4ddc000-e6a3-11e9-93b9-182499c5b357.png">

Also, the Full Blast profile has no effect and the fan profile remains in auto mode.

<img width="933" alt="Screen Shot 2019-10-04 at 12 43 24" src="https://user-images.githubusercontent.com/8654883/66205248-d7bca400-e6a4-11e9-8a5e-8eb962c1b703.png">

```
Macs Fan Control 1.5.0 [Qt 5.6.3]
MacBookPro12,1
CPU: Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz [2 cores]
GPU: Intel Iris Graphics 6100
macOS 10.13.6

Fans:
Right Side-min{1299}curr{1291}max{6199}mode{0}


TempSensors:
TA0P (Ambient) - 42, 42
TB0T (Battery Max) - 36.2969, 36.2969
TB1T (Battery Sensor 1) - 36.2969, 36.2969
TB2T (Battery Sensor 2) - 33.5, 33.5
TBXT () - 36.2969, 36.2969
TC0C (CPU Core 1) - 59, 59
TC0E () - 65.9297, 65.9297
TC0F () - 67.957, 67.957
TC0P (CPU Proximity) - 54.625, 54.625
TC1C (CPU Core 2) - 59, 59
TCGC (GPU PECI) - 60, 60
TCSA () - 59, 59
TCXC (CPU PECI) - 61.0938, 61.0938
TH0A () - 41.625, 41.625
TH0B () - 42.125, 42.125
TH0C () - 42.125, 42.125
TH0F () - 214.125, 214.125
TH0R () - 214.125, 214.125
TH0V () - 41.1602, 41.1602
TH0a () - 41.625, 41.625
TH0b () - 42.125, 42.125
TH0c () - 42.125, 42.125
TH0x () - 42.125, 42.125
THSP () - 129, 129
TM0P (Memory Proximity) - 48.625, 48.625
TMLB () - 129, 129
TPCD (Platform Controller Hub Die) - 54, 54
TW0P () - 129, 129
Th1H (Heatsink Proximity) - 45.875, 45.875
Ts0P (Palm Rest) - 34.5, 34.5
Ts0S () - 41.4531, 41.4531
Ts1P () - 33.5625, 33.5625
Ts1S () - 41.0195, 41.0195
Ts2S () - 38.875, 38.875


Disk drives:
APPLE SSD SM0128G - 42

Path: /Applications/Macs Fan Control.app/Contents/MacOS/Macs Fan Control
```

**Comment by kleuter** [comment]
Thank you, in progress.

**Comment by kleuter** [comment]
Fixed in the current version 1.5.0.4
